src/Personal/reducer.js

```diff
@@ -0,0 +1,33 @@
+import { actionType } from './actionCreator';
+
+const initialUser = {
+  username: '',
+  email: '',
+  accessToken: null,
+};
+
+const user = (state = initialUser, action) => {
+  switch (action.type) {
+    case actionType.SIGNUP_START:
+      return {
```

Comment:
I would suggest to alway place '...state' in return object, it would make adding new fields easier later